### ISSUE TITLE
streamhentaimovies.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#3837]
### Basic Information

**Test link:** `http://streamhentaimovies.com/yokujou-bazooka-the-animation-episode-1/`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.44 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.142

### Additional Message

redirect to scam

**Comment by jspenguin2017** [comment]
Redirect on first load, but the site is blocked by Google Safe Browsing. Putting on hold for now.
